README.md

```diff
@@ -107,3 +94,30 @@ Tagging a release version will:
     - tag as v1.3.1
  - Pushing the update release branch `release-v1.3`
  - Pushing tag `v1.3.1`
+
+## Options
+
+### upstream
+Type: `String`
+Default value: `origin`
+
+If you don't use origin as the name of your upstream remote you can specify
+another, e.g. `git cactus cut --upstream upstream`
+
+### version
+Type: `Boolean`
+Default value: `false`
+
+Show version number of current directory.
```

Comment:
Does this work? I think this shows the version of git-cactus tool right?

Reply:
Ah you're right. I thought it was the current version of the dir you were in... Whoops. I'll edit this.